joern-cli/frontends/javasrc2cpg/src/test/scala/io/joern/javasrc2cpg/querying/OffsetTests.scala

```diff
@@ -90,19 +90,12 @@ class OffsetTests extends JavaSrcCode2CpgFixture {
     ).withConfig(contentEnabled)
 
     "have the first method retrievable via the offset fields" in {
-      def method    = cpg.method.name("method1").head
```

Comment:
wouldn't it make sense to keep the old tests as well? both are valid access paths, no?

Reply:
That's true! I'll split them up and test both